### PR TITLE
[102] [I29SBC]Support select ArrayType in Elastic Search connector

### LINKED ISSUE #102: olk fails to select Array type in Elasticsearch
### Software Environment:
- OpenLooKeng version (source or binary):

- OS platform & distribution (eg., Linux Ubuntu 16.04): Linux

- Java version:
### Describe the current behavior
1.  create an index in Elasticsearch 7.10 containing array data(like "songs" in the picture)
2.  select from the index above, olk will throw error.
154027_cfac562b_1404593

### Describe the expected behavior
select of array data should work

### Steps to reproduce the issue
1. 
image
do this in kibana ui with http://localhost:5601/app/dev_tools#/console
2. Now run select query in olk 
image

### Special notes for this issue

### PR DESCRIPTION
**Pull request by prasad21111995**
### What type of PR is this?

/kind bug

### What does this PR do / why do we need it:
It supports select Array Type in Elastic Search Connector in OLK

### Which issue(s) this PR fixes:

Fixes #102   #I29SBC

### Special notes for your reviewers:

### CODE REVIEW

File: presto-elasticsearch/src/main/java/io/prestosql/elasticsearch/decoders/ArrayDecoder.java [hunk 1]
```diff
@@ -50,7 +43,9 @@ else if (data instanceof List) {
             output.closeEntry();
         }
         else {
-            throw new PrestoException(TYPE_MISMATCH, format("Expected list of elements for field '%s' of type ARRAY: %s [%s]", path, data, data.getClass().getSimpleName()));
+            BlockBuilder array = output.beginBlockEntry();
+            elementDecoder.decode(hit, () -> data, array);
+            output.closeEntry();
```
Comment (Nitin-Kashyap) [inline presto-elasticsearch/src/main/java/io/prestosql/elasticsearch/decoders/ArrayDecoder.java:48]: What type is observed here during your investigation?

what about unsuported type? why should we not have handling for that?

File: presto-elasticsearch/src/main/java/io/prestosql/elasticsearch/decoders/ArrayDecoder.java [hunk 2]
```diff
@@ -13,26 +13,19 @@
  */
 package io.prestosql.elasticsearch.decoders;
```
Comment (Nitin-Kashyap) [inline presto-elasticsearch/src/main/java/io/prestosql/elasticsearch/decoders/ArrayDecoder.java:14]: Add a testcase for the issue.

File: presto-elasticsearch/src/main/java/io/prestosql/elasticsearch/decoders/ArrayDecoder.java (hunk 1 above)
Comment (prasad21111995) [reply presto-elasticsearch/src/main/java/io/prestosql/elasticsearch/decoders/ArrayDecoder.java:48]: getting error like Expected a string or numeric value for field 'songs' of type VARCHAR: [hello, hi, ala] [ArrayList]...now its fixed

Comment (Nitin-Kashyap) [reply presto-elasticsearch/src/main/java/io/prestosql/elasticsearch/decoders/ArrayDecoder.java:48]: Unsupported types should give exception. handle this specific case and add else case which raises exception

**Review by Nitin-Kashyap — CHANGES_REQUESTED** [review]

**Pull request closed without merging**